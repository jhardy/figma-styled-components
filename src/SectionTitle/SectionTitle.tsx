import styled from 'styled-components'
import { Text } from '../Text'

export const SectionTitle = styled(Text)`
  display: flex;
  align-items: center;
  height: 32px;
  color: rgba(0, 0, 0, 0.8);
  background-color: #ffffff;
  font-weight: 600;
`
