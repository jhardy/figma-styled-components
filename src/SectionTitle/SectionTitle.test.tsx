import 'jest-styled-components'
import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { SectionTitle } from './SectionTitle'

test('A default SectionTitle', () => {
  const tree = renderer.create(<SectionTitle>Hello</SectionTitle>).toJSON()
  expect(tree).toMatchSnapshot()
})


