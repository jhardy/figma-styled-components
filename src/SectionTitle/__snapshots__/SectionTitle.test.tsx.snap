// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A default SectionTitle 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
  font-family: "Inter",sans-serif;
  color: rgba(0,0,0,0.8);
  font-weight: 400;
  font-size: 11px;
  -webkit-letter-spacing: 0.005em;
  -moz-letter-spacing: 0.005em;
  -ms-letter-spacing: 0.005em;
  letter-spacing: 0.005em;
  line-height: 16px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 32px;
  color: rgba(0,0,0,0.8);
  background-color: #ffffff;
  font-weight: 600;
}

<span
  className="sc-AxjAm c0"
>
  Hello
</span>
`;
